dashboard/test/models/concerns/user/email_validations_test.rb

```diff
@@ -0,0 +1,124 @@
+require 'test_helper'
+
+class EmailValidationsTest < ActiveSupport::TestCase
+  include Minitest::RSpecMocks
+
+  context 'when building a new user without a password' do
```

Comment:
There are a few test names (here and below) that have `password` in them, is this meant to be `email`?

Reply:
Oh no, I may have mixed up tests with the `PasswordValidations` concern. I'll double check the test are the right ones.